repose-aggregator/services/datastore/impl/src/main/java/org/openrepose/core/services/datastore/impl/ehcache/EHCacheDatastore.java

```diff
@@ -66,6 +66,7 @@ public void put(String key, Serializable value) {
 
     @Override
     public void put(String key, Serializable value, int ttl, TimeUnit timeUnit) {
+        // todo: does not allow for eternal caching (need to either setEternal(true) or set ttl and tti to 0
```

Comment:
Should I create a story for this? It's something that EHCache supports, so we should probably support it as well.